options.go

```diff
@@ -27,6 +27,11 @@ type Options struct {
 
 	// BytesPerSync specifies the number of bytes to write before calling fsync.
 	BytesPerSync uint32
+
+	// Watchable whether to enable event change notification.
+	Watchable bool
+	// WatchQueueSize the cache length of the watch queue, default 1000.
```

Comment:
Can we just use one option WatchQueueSize, if the size is greater than 0, which means enable the watch?